src/nni_manager/training_service/kubernetes/kubeflow/kubeflowApiClient.ts

```diff
@@ -34,12 +34,16 @@ abstract class KubeflowOperatorClient extends KubernetesCRDClient{
                 return new TFOperatorClientV1Alpha2();
             } else if(operatorApiVersion == 'v1beta1') {
                 return new TFOperatorClientV1Beta1();
+            } else if(operatorApiVersion == 'v1beta2') {
+                return new TFOperatorClientV1Beta2();
             }
         } else if(kubeflowOperator === 'pytorch-operator') {
             if(operatorApiVersion == 'v1alpha2') {
                 return new PytorchOperatorClientV1Alpha2();
             } else if(operatorApiVersion == 'v1beta1') {
                 return new PytorchOperatorClientV1Beta1();
+            } else if(operatorApiVersion == 'v1beta2') {
+                return new PytorchOperatorClientV1Beta2();
             }
         }
```

Comment:
while you have 3 'else if', considering refactoring it using dict.

Reply:
refactor to use switch...case...